projects/auth0-angular/src/lib/auth.service.ts

```diff
@@ -118,7 +142,9 @@ export class AuthService implements OnDestroy {
       iif(
         () => isCallback,
         this.handleRedirectCallback(),
-        defer(() => this.auth0Client.checkSession())
+        defer(() => this.auth0Client.checkSession()).pipe(
+          mergeMap(() => this.getAccessTokenSilently(/* { cacheOnly: true } */))
```

Comment:
The SPAJS SDK doesnt support this yet. To avoid unnecessary calls we need a way to get a token from the cache only, without triggering a request.